plan/common_plans.go

```diff
@@ -441,11 +441,11 @@ func (e *Explain) explainPlanInRowFormat(p PhysicalPlan, taskType, indent string
 }
 
 // prepareOperatorInfo generates the following information for every plan:
-// operator id, task type, operator info, and the estemated row count.
-func (e *Explain) prepareOperatorInfo(p PhysicalPlan, taskType string, indent string, isLastChild bool) {
+// operator id, task type, operator info, and the estemated row RowCount.
```

Comment:
`RowCount` -> `count`. And most of the `count` in the comment should not be changed to `RowCount`.

Reply:
To simplify the code movement, I exported the `count` field to `Count`, but the `StatsInfo` struct already has a function `Count()`. Because this simply variable renaming is not allowed, I changed the `count` to `RowCount`.

The comment are changed back to `count` according to the context.